### ISSUE TITLE
autoscaling options not available for default nodepool in google_container_cluster resource

### ISSUE THREAD

**Issue by drzero42** [terraform-providers/terraform-provider-google#473]
### Terraform Version
- Terraform: v0.10.6
- Google Provider: v0.1.3

### Affected Resource(s)
- google_container_cluster
- google_container_node_pool

### Description
It is possible to setup autoscaling on node pools created with google_container_node_pool resource, but this option isn't available for the default pool created as part of the google_container_cluster resource.

**Comment by daaain** [comment]
Just to clarify, this issue means that the code below (with all other properties omitted) is not possible, right?

```
resource "google_container_cluster" "cluster" {
  node_pool {
      autoscaling {
        min_node_count = 1
        max_node_count = 5
      }
    }
  }
}
```

**Comment by danawillow** [comment]
Oh for some reason GitHub didn't auto close this one, but it was fixed in #489 so that code will work once the next release is out.

**Comment by daaain** [comment]
That is brilliant, thank you so much @danawillow 🥇
